dosa.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pani Puri</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<nav>
+    <div class="nav-bar">
+        <h1>Chef Curry's Kitchen</h1>
+        <ul class="nav-elements">
+            <a href="index.html">
+            <li>Home</li>
+            </a>
+            <li>About</li>
+        </ul>
+    </div> 
+</nav>
+<body>
+    <div class="pani-puri-card">
+        <h1>Pani-Puri</h1>
+        <img class="pani-puri-pic" src="images/dosa.webp" alt="Pani-Puri">
+        <div class="label-and-ingredients">
+        <h2 class="ingredient">Ingriedients</h2>
+        <ul class="ingredients">
+            <li>3/4 cup Parboiled Rice (idli-dosa rice)</li>
+            <li>3/4 cup Regular Rice</li>
+            <li>1/2 cup Whole Urad Dal (without skin) or Split Urad Dal (black lentils)</li>
+            <li>1/4 teaspoon Fenugreek Seeds (methi dana)</li>
+            <li>1/2 tablespoon Chana Dal (gram lentils), optional</li>
+            <li>Water, as needed</li>
+            <li>Salt to taste</li>
+            <li>Oil, for shallow frying</li>
+            </ul>
+        </div>
```

Comment:
The indenting here is not great, usually we want all of the child elements of a parent element to be indented one tab further than the parent element. So for every div its children should be on tab in from the div, same for the unordered list and all of its children list elements. You can install the Prettier plugin for VSCode which will auto-format your code so you don't have to worry about these things.